### PR TITLE
kn member photo

### LINKED ISSUE #83: Add member image for Keagan

### PR DESCRIPTION
**Pull request by ctc-devops**
closes #83

### CODE REVIEW

**Review by Vakhshoori101 — COMMENTED** [review]
Everything looks good. However, can you upload a new image that is 260x260 please :)

**Comment by Vakhshoori101** [comment]
Thanks for updating. Last thing is can you fix the merge conflict please. Lmk if you need help :)

File: website/src/app/views/about/members.js [hunk 1]
```diff
@@ -59,6 +59,8 @@ import ted from '../../../images/about/memberImages/ted.jpg';
 import phil from '../../../images/about/memberImages/phil.jpg';
 import kory from '../../../images/about/memberImages/kory.jpg';
 import charles from '../../../images/about/memberImages/charles.jpg';
+import keagan from '../../../images/about/memberImages/keagan.jpg';
+
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:63]: Can you remove this blank line pls :) Then should be good to go.

Comment (keagan-n) [reply website/src/app/views/about/members.js:63]: Got it :)

**Review by Vakhshoori101 — APPROVED** [review]
LGTM

**Pull request merged**